src/ImpactEvaluator.sol

```diff
@@ -1,13 +1,16 @@
 // SPDX-License-Identifier: (MIT or Apache-2.0)
 
+// Keep up to date with ImpactEvaluatorNativeAddr.sol!
+
 import "../lib/openzeppelin-contracts/contracts/access/AccessControl.sol";
 pragma solidity ^0.8.19;
 
 contract ImpactEvaluator is AccessControl {
     struct Round {
         uint end;
         string[] measurementsCids;
-        mapping (address => uint64) scores;
+        address[] participants;
```

Comment:
Is there a reason why we're switching the data structure of participants and scores here?

Reply:
I don't recall the exact reason, but a `mapping (commonTypes.FilAddress => uint64)` didn't work here. I'll try to reproduce later. Is that type an invalid map key type?